Cargo.toml

```diff
@@ -24,4 +24,4 @@ default = []
 self-test = []
 
 [dependencies]
-litrs = "0.2.3"
+litrs = { version = "0.2.3", default-features = false }
```

Comment:
`litrs` comes with [the `proc-macro2` feature enabled by default](https://docs.rs/litrs/latest/litrs/#crate-features), but we don't need it.